src/services/export.js

```diff
@@ -297,14 +317,66 @@ const scrape = async ({ url, format }) => {
     });
 
     Logger.debug('visiting page');
+
+    let auth = AUTH_TYPE_ANONYMOUS;
+    // this endpoint will return a 401 so we catch to find the auth type
+    try {
+      const loginTypeResponse = await request({
+        uri: loginTypeUrl,
+        json: true,
+      });
+      ({ body: { auth } = {} } = loginTypeResponse);
+    } catch (err) {
+      ({ error: { auth } = {} } = err);
+    }
+    console.log(auth);
+
     await page.goto(url, {
       waitUntil: 'networkidle0',
       // one minute timeout
-      timeout: 60000,
+      timeout: TIMEOUT,
     });
 
+    switch (auth) {
+      case AUTH_TYPE_USERNAME:
+        // TODO throw error if no username
+        await page.type('#username', username);
```

Comment:
You can refactor this whole bit (which repeates three times):

```
    await Promise.all([
      page.waitForNavigation({
        timeout: TIMEOUT,
        waitUntil: 'networkidle0',
      }),
      page.click('.submit'),
    ]);
```

To a helper function:

```
const signIn = async (page) => {
  return Promise.all([
      page.waitForNavigation({
        timeout: TIMEOUT,
        waitUntil: 'networkidle0',
      }),
      page.click('.submit'),
    ]);
}
```

And then call it inside each case: `await signIn(page);`